inference-engine/tests/functional/inference_engine/local_test.cpp

```diff
@@ -205,40 +226,23 @@ class LocaleTests : public ::testing::Test {
     void testBody(bool isLSTM = false) const {
         InferenceEngine::Core core;
 
-        // This model contains layers with float attributes.
-        // Conversion from string may be affected by locale.
         std::string model = isLSTM ? _model_LSTM : _model;
-        auto blob = make_shared_blob<uint8_t>(TensorDesc(Precision::U8, {3360}, Layout::C));
+        auto blob = make_shared_blob<uint8_t>(TensorDesc(Precision::U8, {26000000}, Layout::C));
         blob->allocate();
         auto net = core.ReadNetwork(model, blob);
 
-        IE_SUPPRESS_DEPRECATED_START
-        if (!isLSTM) {
-            auto power_layer = dynamic_pointer_cast<PowerLayer>(CommonTestUtils::getLayerByName(net, "power"));
-            ASSERT_EQ(power_layer->scale, 0.75f);
-            ASSERT_EQ(power_layer->offset, 0.35f);
-            ASSERT_EQ(power_layer->power, 0.5f);
-
-            auto sum_layer = dynamic_pointer_cast<EltwiseLayer>(CommonTestUtils::getLayerByName(net, "sum"));
-            std::vector<float> ref_coeff{0.77f, 0.33f};
-            ASSERT_EQ(sum_layer->coeff, ref_coeff);
+        auto convertedNetwork = std::make_shared<InferenceEngine::details::CNNNetworkImpl>(net);
 
-            auto info = net.getInputsInfo();
-            auto preproc = info.begin()->second->getPreProcess();
-            ASSERT_EQ(preproc[0]->stdScale, 0.1f);
-            ASSERT_EQ(preproc[0]->meanValue, 104.006f);
+        if (!isLSTM) {
+            auto roialign_layer = CommonTestUtils::getLayerByName(InferenceEngine::CNNNetwork(convertedNetwork), "output");
+            ASSERT_EQ(roialign_layer->GetParamAsString("pooled_h"), "7");
+            ASSERT_EQ(roialign_layer->GetParamAsString("pooled_w"), "7");
+            ASSERT_EQ(roialign_layer->GetParamAsString("sampling_ratio"), "2");
+            ASSERT_EQ(roialign_layer->GetParamAsFloat("spatial_scale"), 0.25f);
         } else {
-            InferenceEngine::NetPass::UnrollRNN_if(net, [] (const RNNCellBase& rnn) -> bool { return true; });
-            auto lstmcell_layer = dynamic_pointer_cast<ClampLayer>(CommonTestUtils::getLayerByName(net, "LSTMCell:split_clip"));
-
-            float ref_coeff = 0.2f;
-            ASSERT_EQ(lstmcell_layer->min_value, -ref_coeff);
-            ASSERT_EQ(lstmcell_layer->max_value,  ref_coeff);
-
-            ASSERT_EQ(lstmcell_layer->GetParamAsFloat("min"), -ref_coeff);
-            ASSERT_EQ(lstmcell_layer->GetParamAsFloat("max"),  ref_coeff);
+            auto lstmcell_layer = CommonTestUtils::getLayerByName(InferenceEngine::CNNNetwork(convertedNetwork), "LSTMCell");
```

Comment:
`getLayerByName` is deprecated API. We must use ngraph w/o conversion to `CNNNetworkImpl`